src/core/connection.c

```diff
@@ -5276,6 +5293,15 @@ QuicConnRecvFrames(
             AckImmediately = TRUE;
             break;
 
+        case QUIC_FRAME_RELIABLE_RESET_STREAM:
+            QuicTraceLogConnInfo(
+                ReliableResetFrameReceived,
+                Connection,
+                "Received a reliable reset frame. Local reliable reset extension preferences: %hhu",
+                Connection->Settings.ReliableResetEnabled);
```

Comment:
Please remove this log. We have a different way of logging per-frame info (in packet.c).

Reply:
Figured it out, just remove the break so the branch falls through to get rid of the warning lol